src/components/Datepicker/DatepickerRangeInput.tsx

```diff
@@ -224,6 +239,10 @@ const DatepickerRangeInput: FC<DatepickerRangeInputProps> = ({
     disabled,
     ...rest
 }: DatepickerRangeInputProps) => {
+    const [triggerReference, setTriggerReference] = React.useState(undefined);
```

Comment:
Follow the codestyle for this component, please (add `useState` to imports). 

```suggestion
    const [triggerReference, setTriggerReference] = useState(undefined);
```